src/helpers/schema.sql

```diff
@@ -2,6 +2,7 @@ CREATE TABLE `keys` (
   `key` VARCHAR(256) DEFAULT NULL,
   owner VARCHAR(64) NOT NULL,
   name VARCHAR(32) NOT NULL,
+  level VARCHAR(32) NOT NULL DEFAULT 'user',
```

Comment:
can we use an enum type here? 
https://dev.mysql.com/doc/refman/8.0/en/enum.html

Reply:
For what reason?

Reply:
ENUM restricts data to a fixed set of values, whereas VARCHAR does not. This can help ensure data integrity by only allowing predefined values to be stored in the column.

Reply:
Its likely that we will add others choice here that we dont know yet, i'd rather not use enum for this reason, but we could use INT with default to "0" (free tier)

Reply:
Thanks for the question @Todmy and thanks for clarifying @bonustrack ❤️ this clears all the confusion around naming 😅